avatar.css

```diff
@@ -0,0 +1,104 @@
+.avatar_container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    gap: 20px;
+}
+
+.avatar img {
+    vertical-align: middle;
+    width: 100%;
+    height: auto;
+}
+
+.avatar_rd img {
+    border-radius: 50%;
+}
+
+.avatar_lg {
+    width: 10rem;
+    height: 10rem;
+    font-size: 3rem;
+}
+
+.avatar_md {
+    width: 7rem;
+    height: 7rem;
+    font-size: 2rem;
+}
+
+.avatar_sm {
+    width: 4rem;
+    height: 4rem;
+    font-size: 1rem;
+}
+
+.avatar_xs {
+    width: 2.5rem;
+    height: 2.5rem;
+    font-size: 1rem;
+}
+
+.avatar_text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: red;
```

Comment:
Suggestion: Please use hex code for consistency.

Reply:
Done